### ISSUE TITLE
Windows 11, CUDA 12, Kaolin and Omniverse and RTX 4090

### ISSUE THREAD

**Issue by albertotono** [NVIDIAGameWorks/kaolin#676]
What version of CUDA Pytorch would you recommend (11.3, 11.6 or 11.1) for Installing Kaolin on Windows 11? 

Does Kaolin and Omniverse work well with Windows 11/10 also for training? It looks it is supported now for Windows. 
Will it work also in a 4090 or I should use a A6000?

Thanks for the support and reply in advance.

**Comment by Caenorst** [comment]
Hi @albertotono , testing Kaolin on windows is always a bit difficult, however it should work on both Windows on both 11.3 and 11.6. AFAIK it should work on 4090, and if there is any issue on this GPU i will prioritize fixing it.

**Comment by albertotono** [comment]
I ran into several issues with both Linux and Windows with the 4090 tbh. I was able to install Kaolin with my previous 2080 Ti, and with a A5000 but not on the 4090 unfortunately. I also had this issue with Kaolin/Omniverse on Linux https://forums.developer.nvidia.com/t/gpu-rtx4090-v-s-rtx-3090/236867/6 . Was someone able to run with a 4090 on windows? 

I am closing it for now, since you confirm that it should work. So I will open a new issue with the error attach as soon as I will get back to it.
